tests/test_integration.py

```diff
@@ -233,10 +233,18 @@ def test_send_presampler_headers(self, mocked_http):
         eq_(request_call.call_count, 1)
 
         # retrieve the headers from the mocked request call
+        expected_headers = {
+                'Meta-Lang': 'python',
```

Comment:
now that I think about it, I'd say to follow what we already have, so keeping a prefix for our headers:
```python
expected_headers = {
    'Datadog-Meta-Lang': 'python',
    'Datadog-Meta-Lang-Interpreter': PYTHON_INTERPRETER,
    'Datadog-Meta-Lang-Version': PYTHON_VERSION,
    'X-Datadog-Trace-Count': '1',
    'Content-Type': 'application/msgpack'
}
```

I'd say to not add `X-` because it has been deprecated since 2012: https://tools.ietf.org/html/rfc6648 and not having it is still a best practice. Though I'd say to keep `X-Datadog-Trace-Count` (for now) because we have that in the agent and it's tough to keep the client and the agent in sync.